vernac/comFixpoint.ml

```diff
@@ -366,18 +366,18 @@ let check_safe () =
   let flags = Environ.typing_flags (Global.env ()) in
   flags.check_universes && flags.check_guarded
 
-let do_fixpoint ~ontop local poly l =
+let do_fixpoint ~is_ontop local poly l =
```

Comment:
Umm, `is_ontop` should be more like `is_nested`, right?

Reply:
Changed